app/assets/stylesheets/aspects.scss

```diff
@@ -4,20 +4,27 @@
     @include selectable-list;
 
     .status_indicator {
-      width: 19px;
       height: 14px;
+      width: 19px;
       display: inline-block;
     }
 
     a {
       .text {
-        color: #333333;
+        $color: #333;
       }
+
       &:hover {
         background: $background-grey;
       }
+
       cursor: pointer;
       padding-left: 10px;
+
     }
   }
 }
+.aspect_membership {
+  height: 300px;
```

Comment:
That won't work, as if you don't have much aspects, the list will now always be at least 300px large. Please use `max-height` instead.